packages/kbn-managed-vscode-config-cli/index.ts

```diff
@@ -34,7 +34,7 @@ run(async ({ log }) => {
     }
   }
 
-  const updatedJson = updateVscodeConfig(
+  const updatedJson = await updateVscodeConfig(
```

Comment:
This value is passed to `Fsp.writeFile` below which but be a `string` but this _may_ return `Promise<string>`. See bootstrap error below.

```
 ERROR  [bootstrap] failed: [node] exitted with 1:
          output:
            ERROR UNHANDLED ERROR
            ERROR TypeError: The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received an instance of Promise
                      at Object.writeFile (node:internal/fs/promises:1206:5)
                      at index.ts:54:13
                      at run.ts:73:10
                      at withProcRunner (with_proc_runner.ts:29:5)
                      at run (run.ts:71:5)
error Command failed with exit code 1.
```